axelor-base/src/main/java/com/axelor/apps/base/service/print/PrintTemplateLineServiceImpl.java

```diff
@@ -65,8 +64,7 @@ public String checkExpression(
     String resultOfContent = null;
 
     Locale locale =
-        Optional.ofNullable(printTemplateLine.getPrintTemplate().getLanguage())
-            .map(Language::getCode)
+        Optional.ofNullable(printTemplateLine.getPrintTemplate().getLocalization().getCode())
```

Comment:
Keep the .map(Localization::getCode), otherwise there is no point to use the Optional.ofNullable, you'll get an NPE if localization is null